packages/node/test/fixtures/62-edge-middleware/middleware.ts

```diff
@@ -0,0 +1,12 @@
+export default (req: Request) => {
+  const url = new URL(req.url);
+  if (url.pathname === '/' || url.pathname.startsWith('/api/')) {
+    // For `index.html` and `/api/edge.js`, pass through
+    const response = new Response();
+    response.headers.set('x-middleware-next', '1');
```

Comment:
It would be good to add a header here to confirm that middleware was actually executed (since default behavior is pass-through)

Reply:
653bb8f8da4f4fa50c1bc5ec8baf8aeafd43dd95